optimum/intel/openvino/configuration.py

```diff
@@ -33,18 +33,17 @@
 logger = logging.getLogger(__name__)
 
 _DEFAULT_4BIT_CONFIGS = {
-    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "ratio": 0.8},
+    "databricks/dolly-v2-3b": {"bits": 4, "sym": False, "group_size": 128, "scale_estimation": True},
     "EleutherAI/gpt-j-6b": {"bits": 4, "sym": False, "group_size": 64},
     "facebook/opt-6.7b": {"bits": 4, "sym": False, "group_size": 64, "ratio": 0.8},
-    "bigscience/bloomz-7b1": {"bits": 4, "sym": False, "group_size": 32, "ratio": 0.6},
```

Comment:
Is it removed because the configuration is no more adapted ?

Reply:
Yes. Since an asymmetric scheme is the default, it would be enough for this model and custom configuration is no longer needed.